_i18n/nb-no/resources/moneropedia/randomx.md

```diff
@@ -0,0 +1,11 @@
+---
+terms: ["randomx", "random-x"]
+summary: "ASIC-resistent POW-algoritme som for tiden brukes av Monero"
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+RandomX er en «Proof Of Work»-algoritme, utviklet av Monero-bidragsytere og tatt i bruk av Monero fra og med 0.15-utgivelsen.
```

Comment:
Maybe change to "...Monero-bidragsytere, og ble tatt i bruk av Monero fra og med 0.15-utgivelsen."